Shared/Samples/Add clustering feature reduction to a point feature layer/AddClusteringFeatureReductionToAPointFeatureLayerView.swift

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct AddClusteringFeatureReductionToAPointFeatureLayerView: View {
+    /// The model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The popup to be shown as the result of the layer identify operation.
+    @State private var popup: Popup?
+    
+    /// A Boolean value specifying whether the popup view is shown.
```

Comment:
```suggestion
    /// A Boolean value indicating whether the popup view is shown.
```

Not a big deal but _indicating_ is generally used here.